client/src/test/runTest.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import { runTests } from 'vscode-test';
+
+async function main() {
+    try {
+        // The folder containing the Extension Manifest package.json
+        // Passed to `--extensionDevelopmentPath`
+        const extensionDevelopmentPath = path.resolve(__dirname, '../../../');
+
+        // The path to the extension test runner script
+        // Passed to --extensionTestsPath
+        const extensionTestsPath = path.resolve(__dirname, './suite/index');
```

Comment:
Probably ⇔ and less confusing:

```suggestion
        const extensionTestsPath = path.resolve(__dirname, 'suite/index');
```

Check if you will.